tutorial-running-chaos-workflow-on-remote-cluster/your-first-pwapp/index.html

```diff
@@ -0,0 +1,369 @@
+</td></tr>
+<tr><td colspan="1" rowspan="1"><p>litmusctl-windows-arm64</p>
+</td><td colspan="1" rowspan="1"><p><a href="https://litmusctl-bucket.s3-eu-west-1.amazonaws.com/litmusctl-windows-arm64-v0.3.0.tar.gz" target="_blank">Click here</a></p>
+</td><td colspan="1" rowspan="1"><p><a href="https://litmusctl-bucket.s3-eu-west-1.amazonaws.com/litmusctl-windows-arm64-v0.2.0.tar.gz" target="_blank">Click here</a></p>
+</td><td colspan="1" rowspan="1"><p><a href="https://litmusctl-bucket.s3-eu-west-1.amazonaws.com/litmusctl-windows-arm64-master.tar.gz" target="_blank">Click here</a></p>
+</td></tr>
+<tr><td colspan="1" rowspan="1"><p>litmusctl-windows-arm</p>
+</td><td colspan="1" rowspan="1"><p><a href="https://litmusctl-bucket.s3-eu-west-1.amazonaws.com/litmusctl-windows-arm-v0.3.0.tar.gz" target="_blank">Click here</a></p>
+</td><td colspan="1" rowspan="1"><p><a href="https://litmusctl-bucket.s3-eu-west-1.amazonaws.com/litmusctl-windows-arm-v0.2.0.tar.gz" target="_blank">Click here</a></p>
+</td><td colspan="1" rowspan="1"><p><a href="https://litmusctl-bucket.s3-eu-west-1.amazonaws.com/litmusctl-windows-arm-master.tar.gz" target="_blank">Click here</a></p>
+</td></tr>
+</table>
+<h3 is-upgraded>Linux/MacOS</h3>
+<h3 is-upgraded><strong>Extract the binary</strong></h3>
+<pre><code>tar -zxvf litmusctl-&lt;OS&gt;-&lt;ARCH&gt;-&lt;VERSION&gt;.tar.gz</code></pre>
+<h3 is-upgraded><strong>Provide necessary permissions</strong></h3>
+<pre><code>chmod +x litmusctl</code></pre>
+<h3 is-upgraded><strong>Move the litmus binary to /usr/local/bin/litmusctl. </strong></h3>
+<h3 is-upgraded>Note: Do this using root privileges</h3>
+<pre><code>sudo mv litmusctl /usr/local/bin/litmusctl</code></pre>
+<h3 is-upgraded>Windows</h3>
+<h3 is-upgraded><strong>Extract the binary</strong></h3>
+<pre><code>litmusctl.exe &lt;command&gt; &lt;subcommand&gt; &lt;subcommand&gt; [options and parameters]</code></pre>
+<h3 is-upgraded><strong>Check the litmusctl version</strong></h3>
+<pre><code>litmusctl version</code></pre>
+
+
+      </google-codelab-step>
+    
+      <google-codelab-step label="Setup an external agent" duration="5">
+        <p>We can now use Litmusctl to create an external agent on the remote cluster. Open your terminal on your local Machine and run the command </p>
+<pre><code>litmusctl agent connect</code></pre>
+<p>You will be prompted to do the following</p>
+<h3 is-upgraded><strong>Login to the remote portal </strong></h3>
+<p>Type in your host IP, username, and password</p>
+<pre><code>🔥Connecting LitmusChaos agent
+📶Please enter LitmusChaos details --
+👉Host URL where litmus is installed: http://34.132.48.214:32543
+🤔Username [admin]: admin
+🙈Password:
+✅Login Successful!</code></pre>
+<h3 is-upgraded><strong>Select a Project</strong></h3>
+<pre><code>✨Projects List:
+1.  admin&#39;s project
+🔎Select Project: 1</code></pre>
+<pre><code>🏃Running prerequisites check....
```

Comment:
Step for selection of agent scope is missing

Reply:
Could this be an issue with the version of litmusctl I used? I didn't see a step for selecting an agent scope.

Reply:
Yes, it may be the issue. Previously there was an issue like this. Can you add that step from latest version.

Reply:
@Jonsy13 The issue I had is that I am using litmusctl v0.2.0 because I am running litmus Beta8. litmusctl v0.2.0. has compatibility issues with versions of litmus higher Beta8, and these version are only compatible with litmusctl v0.3.0.

I knew this might introduce some slight differences, so I added a note in the installation step of the tutorial stating this. 
I attached an image of the note below. 

<img width="1251" alt="Screenshot 2021-09-23 at 7 43 18 AM" src="https://user-images.githubusercontent.com/29985200/134464601-797cb1ca-f98e-4f59-a69e-80e336b4bd1b.png">

Reply:
@rajdas98 can take a look here.